### PR TITLE
add the map for clouds

### PR DESCRIPTION
**Pull request by SIMSEUNGMIN**

### CODE REVIEW

**Comment by robinpaulson** [comment]
This is great work, but there are a few parts missing. Firstly, we need an icon, which should be in here:
https://github.com/martykan/forecastie/tree/master/app/src/main/res/drawable

Also, this needs an entry:
https://github.com/martykan/forecastie/blob/master/app/src/main/res/menu/menu_map_bottom.xml

I added a note to the code too, you deleted a line and I'm not sure why.

File: app/src/main/assets/map.html [hunk 1]
```diff
@@ -66,13 +66,21 @@
...
+        cloudsLayer.setOpacity(0.7);
+         map.addLayer(cloudsLayer);
+
         // Rain
         var rainLayer = new L.TileLayer("http://tile.openweathermap.org/map/precipitation_new/{z}/{x}/{y}.png?appid=" + getUrlParameter('appid'), {
             minZoom: 1,
             maxZoom: 15
         });
         rainLayer.setOpacity(0.7);
-        map.addLayer(rainLayer);
```
Comment (robinpaulson) [inline app/src/main/assets/map.html]: i think this line is necessary

**Comment by robinpaulson** [comment]
These are relatively minor things, which shouldn't be too difficult to fix. We will need to create/adapt an icon though, perhaps there is a glyph in the weather icons font we can use?

**Comment by SIMSEUNGMIN** [comment]
I used icon in ic_cloud_grey600_24dp folder. Isn't this icon in drawable?

File: app/src/main/assets/map.html (hunk 1 above)
Comment (SIMSEUNGMIN) [reply app/src/main/assets/map.html]: okay, I understand. I fix it

Comment (SIMSEUNGMIN) [reply app/src/main/assets/map.html]: this code is added 75 line

**Comment by robinpaulson** [comment]
> I used icon in ic_cloud_grey600_24dp folder at drawable. Isn't this icon in drawable?

oh, i missed that. my bad.

**Comment by robinpaulson** [comment]
It looks good so far, I'll check it out in more detail later in the week

File: app/src/main/assets/map.html (hunk 1 above)
Comment (robinpaulson) [reply app/src/main/assets/map.html]: I don't understand your response. According to Github, you deleted line 75 in the file "map.html". Why, I think it's necessary to have it there?

Comment (robinpaulson) [reply app/src/main/assets/map.html]: Could you address this please @SIMSEUNGMIN ?

Comment (SIMSEUNGMIN) [reply app/src/main/assets/map.html]: In this my code, this code you said is 75 line. 
The reason that I deleted "map.addLayer(rainLayer);" and put new code is the screen shows the cloud map first when you click the map.

File: app/src/main/assets/map.html [hunk 2]
```diff
@@ -66,13 +66,21 @@
         });
         windLayer.setOpacity(0.7);
 
+        // Clouds
+        var cloudsLayer = new L.TileLayer("http://tile.openweathermap.org/map/clouds_new/{z}/{x}/{y}.png?appid=" + getUrlParameter('appid'), {
+            minZoom: 1,
+            maxZoom: 15
+        });
+        cloudsLayer.setOpacity(0.7);
+         map.addLayer(cloudsLayer);
```
Comment (SIMSEUNGMIN) [inline app/src/main/assets/map.html]: This code has same meaning as code you said.

File: app/src/main/assets/map.html (hunk 1 above)
Comment (robinpaulson) [reply app/src/main/assets/map.html]: OK, I'll accept the pull when we're due for a new release in a week or so. Cheers

**Pull request merged**